src/Hangfire.Core/Dashboard/Content/css/hangfire.css

```diff
@@ -217,6 +217,12 @@ a:hover .label-hover {
     margin-bottom: 10px;
 }
 
+.fltr-txtbx{
```

Comment:
are there any examples of how this is named in the rest of the project? just want to make sure we are being consistent with the original project